linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/JSGen.scala

```diff
@@ -262,15 +262,12 @@ private[emitter] final class JSGen(val semantics: Semantics,
     Apply(coreJSLibVar(helperName), args.toList)
   }
 
-  def encodeClassVar(className: ClassName)(implicit pos: Position): Tree =
```

Comment:
I'm not sure about this removal. Yes, it's a simple forwarder, but it's used quite a lot.